rdkafka-sys/src/bindings.rs

```diff
@@ -976,7 +976,7 @@ pub enum rd_kafka_queue_s { }
 pub type rd_kafka_queue_t = rd_kafka_queue_s;
 #[derive(Copy, Clone)]
 #[repr(i32)]
-#[derive(Debug)]
+#[derive(Debug,PartialEq)]
```

Comment:
I added this here manually. Maybe there's an option in Bindgen to generate this? Not sure which command you used.